ldap/servers/slapd/back-ldbm/ldbm_modrdn.c

```diff
@@ -1557,6 +1566,204 @@ ldbm_back_modrdn(Slapi_PBlock *pb)
     return retval;
 }
 
+
+/* dsEntryDN functions */
+
+static int
+dsentrydn_moddn_rename_child(
+    back_txn *ptxn,
+    Slapi_Backend *be,
+    struct ldbminfo *li,
+    struct backentry *e,
+    struct backentry **ec,
+    int parentdncomps,
+    char **newsuperiordns,
+    int newsuperiordncomps)
+{
+    /*
+     * Construct the new DN for the entry by taking the old DN
+     * excluding the old parent entry DN, and adding the new
+     * superior entry DN.
+     */
+    int retval = 0;
+    char *olddn;
+    char *newdn;
+    char **olddns;
+    int olddncomps = 0;
+    int need = 1; /* For the '\0' */
+    size_t i;
+
+    olddn = slapi_entry_attr_get_charptr((*ec)->ep_entry, SLAPI_ATTR_DS_ENTRYDN);
+    if (NULL == olddn) {
+        return retval;
+    }
+    olddns = slapi_ldap_explode_dn(olddn, 0);
```

Comment:
💥

Reply:
??  This explodes?

Reply:
> It's a joke comment. ldap_explode_dn -> boom

I really appreciate that!  Just wasn't expecting it haha

Reply:
Sometimes I'm funny :)